classes/main.rb

```diff
@@ -18,69 +18,69 @@ def display_ui
     Execute.new(user_choice).execute
   end
 
+  # rubocop:disable Metrics/MethodLength
+  # rubocop:disable Metrics/AbcSize
+  # rubocop:disable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
   def load_data
-    display_books
-    display_people
-    display_rentals
-  end
-
-  private
-
-  def load_json_file(filename)
-    return unless File.exist?(filename)
-
-    data = File.read(filename)
-    JSON.parse(data) unless data.empty?
-  end
-
-  def display_books
-    books = load_json_file('books.json')
-    if books
-      puts 'Books:'
-      books.each { |book| display_book(book) }
+    if File.exist?('./class/books.json')
+      books_data = File.read('./class/books.json')
+      if books_data.empty?
+        puts 'No books available.'
+      else
+        books = JSON.parse(books_data)
+        puts 'Books:'
+        books.each do |book|
+          puts "Title: #{book['title']} Author: #{book['author']}"
+        end
+      end
     else
-      puts 'No books available.'
+      puts 'Books file does not exist.'
     end
-  end
 
-  def display_book(book)
-    puts "Title: #{book['title']} Author: #{book['author']}"
-  end
-
-  def display_people
-    people = load_json_file('people.json')
-    if people
-      puts 'People:'
-      people.each { |person| display_person(person) }
+    if File.exist?('./class/people.json')
+      people_data = File.read('./class/people.json')
+      if people_data.empty?
+        puts 'No person available'
+      else
+        people = JSON.parse(people_data)
+        puts 'People:'
+        people.each do |person|
+          puts "ID: #{person['id']} Age: #{person['age']} Name: #{person['name']} Rental: #{person['rental']}"
+        end
```

Comment:
- [ ] Hello @Simpleshaikh1, great effort working on your project but currently, there is something I would kindly suggest you work on to make your project look perfect and ready to go. I notice that whenever person data is preserved in the JSON file, the values given to the parameters are not the same, and only the teacher data is preserved, and the student data are not found in the JSON file.

Below is a video recording of how the app is currently behaving and thing you need to attend to 👇 


https://github.com/Simpleshaikh1/OOP-SCHOOL-LIBRARY-RUBY/assets/78634841/214f32ce-59ab-40d3-9dd9-9cabba19b62c